### ISSUE TITLE
Mobile Navigation

### ISSUE THREAD

**Issue by dhanishgajjar** [ahmadawais/brandfie#4]
It doesn't seem to work!

**Comment by ahmadawais** [comment]
Figure it out. It works otherwise. Did you change anything in the JS or the classes? I need more info here. Do we really need mobile nav?

**Comment by dhanishgajjar** [comment]
Actually I was thinking of getting rid of the entire nav.
It isn’t a long web page.

**Comment by ahmadawais** [comment]
Exactly! Just have the pricing link and how it works! And maybe a contact link!

**Comment by dhanishgajjar** [comment]
So keep the desktop links and no mobile navigation?

**Comment by ahmadawais** [comment]
Three links are easy enough to be displayed on mobile as well as desktop we don't need a folded menu!
